### ISSUE TITLE
srpm v2.0.2 fails to build due to a prte / prrte discrepancy

### ISSUE THREAD

**Issue by anderbubble** [openpmix/prrte#1425]
Thank you for taking the time to submit an issue!

### What version of the PMIx Reference Server are you using? (e.g., v1.0, v2.1, git master @ hash, etc.)

prte-2.0.2-1.src.rpm

##### What version of PMIx are you using? (e.g., v1.2.5, v2.0.3, v2.1.0, git branch name and hash, etc.)

n/a

### Please describe the system on which you are running

* Operating system/version: Rocky Linux 8.6
* Computer hardware: single x86_64 server
* Network type: n/a

-----------------------------

## Details of the problem

Attempting to build `prte-2.0.2-1.src.rpm` on EL8.6 fails, apparently because while the package, tarball, and spec are named "prte," the directory contained in the source tarball is named "prrte."

```shell
$ rpmbuild --rebuild prte-2.0.2-1.src.rpm 
Installing prte-2.0.2-1.src.rpm
warning: user mpiuser does not exist - using root
warning: group mpiuser does not exist - using root
warning: user mpiuser does not exist - using root
warning: group mpiuser does not exist - using root
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.6zShSi
+ umask 022
+ cd /home/ciq/rpmbuild/BUILD
+ rm -rf /home/ciq/rpmbuild/BUILDROOT/prte-2.0.2-1.el8.x86_64
+ cd /home/ciq/rpmbuild/BUILD
+ rm -rf prte-2.0.2
+ /usr/bin/bzip2 -dc /home/ciq/rpmbuild/SOURCES/prte-2.0.2.tar.bz2
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd prte-2.0.2
/var/tmp/rpm-tmp.6zShSi: line 46: cd: prte-2.0.2: No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.6zShSi (%prep)


RPM build errors:
    user mpiuser does not exist - using root
    group mpiuser does not exist - using root
    user mpiuser does not exist - using root
    group mpiuser does not exist - using root
    Bad exit status from /var/tmp/rpm-tmp.6zShSi (%prep)
```

```
$ ls -l /home/ciq/rpmbuild/BUILD
total 0
drwxr-xr-x 8 ciq ciq 286 Feb 10 23:55 prrte-2.0.2
```

**Comment by rhc54** [comment]
Sigh - yeah, we had someone get a little too ambitious with global "search/replace" and cause some problems like this. The correct name for the package is "prrte". It has been a long time since I tried to build from src rpm, but I can give it a shot and try to correct it.

Note that we are about to release v3.0, so I'll make sure that src rpm is correct. Might take me a bit to work back to the 2.0.2 version.

**Comment by rhc54** [comment]
I believe I have this fixed in https://github.com/openpmix/prrte/pull/1426. I will post a src rpm for PRRTE v3 on its release candidate tag if you want to give it a shot, just to ensure things are fixed.

**Comment by rhc54** [comment]
FWIW: I have posted a srpm for the v3.0.0rc1 if you care to test it: https://github.com/openpmix/prrte/releases/download/v3.0.0rc1/prrte-3.0.0-1.src.rpm

**Comment by anderbubble** [comment]
Will do. Thanks!

**Comment by anderbubble** [comment]
Sorry I never tested this before, but I'm back on it now!

**Comment by anderbubble** [comment]
@rhc54 with your changes, and a few of my own in #1512, I can now build this RPM. Thanks for your help!

Can my specfile changes be merged, too?

**Comment by rhc54** [comment]
Definitely, as soon as they pass CI (which is a no-brainer for that change). Thanks for the assist!

FWIW: PRRTE will be requiring a higher version of PMIx when released as v3.0, but I'll update the spec file accordingly.

**Comment by rhc54** [comment]
Closing as completed.
